plugins/modules/mso_schema_site.py

```diff
@@ -123,6 +123,8 @@ def main():
     schema = module.params.get('schema')
     site = module.params.get('site')
     template = module.params.get('template')
+    if template is not None:
+        template = template.replace(' ', '')
```

Comment:
I would put the replace on line 160 like the other occurrences

Reply:
The template is not required, so we need to check whether the template is none before replacing.